src/metadata.rs

```diff
@@ -120,3 +120,45 @@ impl<'a> SstFileMetaData<'a> {
         }
     }
 }
+
+pub struct LiveFiles {
+    inner: *const DBLivefiles,
+}
+
+impl LiveFiles {
+    pub fn from_ptr(inner: *mut DBLivefiles) -> LiveFiles {
+        LiveFiles { inner: inner }
```

Comment:
Our CI doesn't work, probably should run clippy at your end.
```suggestion
        LiveFiles { inner }
```

Reply:
refer to `ColumnFamilyMetaData { inner: inner }`

Reply:
seem that rust-rocksdb lib has not been checked for clippy